src/renderer/entities/wallet/lib/wallet-utils.ts

```diff
@@ -62,6 +64,10 @@ function isWalletConnectGroup(wallet?: Pick<Wallet, 'type'>): wallet is WalletCo
   return isNovaWallet(wallet) || isWalletConnect(wallet);
 }
 
+function isProxiedWallet(wallet?: Pick<Wallet, 'type'>): wallet is ProxiedWallet {
```

Comment:
Other functions don't have `Wallet` suffix, let's stick to that rule
```suggestion
function isProxied(wallet?: Pick<Wallet, 'type'>): wallet is ProxiedWallet {
```